charts/spire/Chart.yaml

```
@@ -38,3 +38,7 @@ maintainers:
     email: marco.franssen@philips.com
     url: https://marcofranssen.nl
 kubeVersion: ">=1.21.0-0"
+dependencies:
+  - name: spiffe-oidc-discovery-provider
+    repository: file:///charts/spiffe-oidc-discovery-provider
```

Maybe this:
```suggestion
    repository: file://./charts/spiffe-oidc-discovery-provider
```
Or alternatively use repo references like
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/Chart.yaml